handler/src/main/java/io/netty/handler/ssl/JdkSslContext.java

```diff
@@ -440,20 +441,44 @@ static JdkApplicationProtocolNegotiator toNegotiator(ApplicationProtocolConfig c
      * @param keyPassword the password of the {@code keyFile}.
      *                    {@code null} if it's not password-protected.
      * @param kmf The existing {@link KeyManagerFactory} that will be used if not {@code null}
+     * @param keyStore the keystore that should be used in the {@link KeyManagerFactory}
      * @return A {@link KeyManagerFactory} based upon a key file, key file password, and a certificate chain.
      * @deprecated will be removed.
      */
     @Deprecated
     protected static KeyManagerFactory buildKeyManagerFactory(File certChainFile, File keyFile, String keyPassword,
```

Comment:
@SplotyCode can you make this package-private and remove the `@deprecated ` tag ?